contracts/crates/chainlink-solana/Cargo.toml

```diff
@@ -15,6 +15,8 @@ name = "chainlink_solana"
 default = []
 
 [dependencies]
-solana-program = "1.17.25"
-borsh = "0.9.1"
-borsh-derive = "0.9.1"
+anchor-lang  = {version = "0.31.1" }
```

Comment:
Is depending on Anchor really necessary? It forces all downstream consumers to depend on Anchor. We deliberately avoided pulling it in when we released the crate

Reply:
Pull in the changes from here: https://github.com/smartcontractkit/chainlink-solana/pull/1271/files